templates/webapi/test/overview.html.ep

```diff
@@ -128,15 +128,15 @@
                 <div class="row" id="filter-modules">
                     <div class="col-4">
                         <div class="form-group">
-                            <label for="filter-modules"><strong>Module</strong></label>
+                            <label for="filter-modules"><strong>Module by name</strong></label>
                             <input name="modules" type="text" id="modules"
                             placeholder="modules, comma separated, e.g. mod1,mod2"
                             class="form-control">
                         </div>
                     </div>
                     <div class="col-4">
                         <div class="form-group">
-                            <label for="filter-module-re"><strong>Test module</strong></label>
+                            <label for="filter-module-re"><strong>Module by content</strong></label>
```

Comment:
Same as above and I suppose "code" is more specific here:
```suggestion
                            <label for="filter-module-re"><strong>Module code</strong></label>
```

Reply:
@Martchus My suggestion was "content" as well. What do you think is more specific about "code" here?